src/shipit_workflow/default.nix

```diff
@@ -1,6 +1,10 @@
 { releng_pkgs
 }:
 
+{ githubCommit ? "unknown"
+, taskGroupId ? null
+}:
+
```

Comment:
this will break other `./please` commands since we expect `default.nix` to return a project derivation and not a function.